airavata_custos_portal/settings.py

```diff
@@ -27,7 +28,7 @@
 )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True #os.environ.get("DJANGO_DEBUG", "true") == "true"
+DEBUG = os.environ.get("DJANGO_DEBUG", "true") == "true"
```

Comment:
We might as well read this from .env as well

Reply:
I think it's good to keep the custos-related variables in the .env. Otherwise, there are more variables for example `DJANGO_STATIC_ROOT` also.

Reply:
Okay, I think that's fine as long as we document it clearly. One other thing I'll add, it's very important that we have DJANGO_DEBUG set to false in production because when DJANGO_DEBUG is true it can leak important sensitive information like Custos client secret, etc.

Reply:
@machristie This PR does not have a production setup as of an ansible or something. So, I'm not sure where to put this. Also, these are Django framework-related and nothing specific to this portal.

Reply:
@dinukadesilva what about the docker-compose.yml file? I guessed that that was what is used for the production deployment, but I could be wrong.

Reply:
Docker compose is the old setup Marcus. I didn't delete them for now because the deployment part is yet to be done.